test/e2e/es/mutation_test.go

```diff
@@ -208,6 +208,25 @@ func TestMutationWithChangeBudget(t *testing.T) {
 	RunESMutation(t, b, mutated)
 }
 
+func TestMutationWithLargerChangeBudget(t *testing.T) {
```

Comment:
I think we are just using a larger `maxUnavailable`, not a larger `maxSurge`, correct? So I wonder if this should be more specific